lib/index.js

```diff
@@ -31,14 +31,17 @@ Sitemon.prototype.send = function(data) {
     }.bind(this));
 };
 
-var formatData = function(type, data) {
-    var message = ['type=' + type];
+var formatData = function(table, keys, values) {
+    var message = ['table=' + table];
 
-    for (var key in data) {
-        message.push(key + '=' + data[key]);
+    for (var key in keys) {
+        message.push('k.' + key + '=' + keys[key]);
+    }
+    for (var key in values) {
```

Comment:
`var key` is used twice in this closure. I suggest using a new variable name for the second loop.